cardano-api/src/Cardano/Api.hs

```diff
@@ -808,6 +810,13 @@ module Cardano.Api (
     parseFilePath,
     parseFileOut,
     parseDirectory,
+
+    toGenesisFileIn,
```

Comment:
These functions aren't used anywhere. Why implement them?

Reply:
For completeness so that if someone wanted to use them, they'd be available.